### PR TITLE
Add files for Rakudo Star

### PR DESCRIPTION
**Pull request by hoelzro**

### CODE REVIEW

File: rakudo-star/content.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# What is Rakudo Star?
+
+Rakudo Star is a Perl 6 distribution designed for use by early adopters of the language.  It includes
+a virtual machine (the JVM or MoarVM), the Rakudo Perl 6 compiler, and a suite of modules that users
+may find useful.  This image includes the MoarVM backend for the compiler.
+
+Project homepage: http://rakudo.org
```
Comment (tianon) [inline rakudo-star/content.md]: The Hub's rendering engine won't linkify these, unfortunately. :cry: (gotta be explicit `[text](url)` references)

Comment (hoelzro) [inline rakudo-star/content.md]: Noted, and fixed!

**Comment by yosifkit** [comment]
LGTM

**Comment by yosifkit** [comment]
Oh, `update.sh` will need a url added to the list :smile:.

**Comment by hoelzro** [comment]
@yosifkit Done!

**Comment by yosifkit** [comment]
@hoelzro, thanks!  LGTM.

ping @tianon

**Comment by tianon** [comment]
LGTM

**Pull request merged**